bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/LedgerEntries.java

```diff
@@ -35,24 +35,11 @@
     LedgerEntry getEntry(long entryId);
 
     /**
-     * In this method, It does not increment the reference counts of ByteBuf for the entries in this LedgerEntries.
+     * This method does not increment the reference count of ByteBuf for the entries in this LedgerEntries.
      * The caller who calls {@link #iterator()} should be careful for not releasing the references.
-     *
-     * when iterator is called, you are handing out the entries, you may not know when the caller will
-     * complete iterating the entries. In this case, please use {@link #retainedIterator()}, it will increment
-     * the refCnt, and the application who is holding iterator should be responsible for releasing one refCnt
-     * that {@link #retainedIterator()} retains for entries.
+     * The implementation of this interface will release all the Entries ByteBuf reference when close.
```

Comment:
Remove this sentence if you add what I put above.

Reply:
will change it.